x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/field_mapping.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiDescribedFormGroup, EuiFormRow, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+import styled from 'styled-components';
+
+import * as i18n from './translations';
+import { FieldMappingRow } from './field_mapping_row';
+
+const FieldRowWrapper = styled.div`
+  margin-top: 8px;
+  font-size: 14px;
+`;
+
+const supportedThirdPartyFields = [
+  {
+    value: 'short_description',
+    label: 'Short Description',
+  },
+  {
+    value: 'comment',
+    label: 'Comment',
+  },
+  {
+    value: 'tags',
+    label: 'Tags',
+  },
+  {
+    value: 'description',
+    label: 'Description',
+  },
+];
+
+const FieldMappingComponent: React.FC = () => {
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      title={<h3>{i18n.FIELD_MAPPING_TITLE}</h3>}
+      description={i18n.FIELD_MAPPING_DESC}
+    >
+      <EuiFormRow fullWidth>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <span className="euiFormLabel">{i18n.FIELD_MAPPING_FIRST_COL}</span>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <span className="euiFormLabel">{i18n.FIELD_MAPPING_SECOND_COL}</span>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <span className="euiFormLabel">{i18n.FIELD_MAPPING_THIRD_COL}</span>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFormRow>
+      <FieldRowWrapper>
+        <FieldMappingRow siemField="Name" thirdPartyFields={supportedThirdPartyFields} />
```

Comment:
do we plan to modify `supportedThirdPartyFields` or it is going to be hardcoded?

Reply:
The idea is to have dynamic mapping by fetching the mapping from ServiceNow but for the moment we do not support that. This component probably will change in the next PRs.